augur/curate/titlecase.py

```diff
@@ -0,0 +1,127 @@
+#!/usr/bin/env python3
+"""
+Standardizes string fields of the NDJSON record from stdin and outputs the
+modified record to stdout.
+"""
+import argparse
+
+import re
+import unicodedata
```

Comment:
Unused import. We should have linting for this, but I guess we don't.

Reply:
Dropped unused import, and fixup'd in https://github.com/nextstrain/augur/pull/1197/commits/c00156ba7d31026aa89fc5ed68ae5215c09b99bd